index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <title></title>
+</head>
+<body>
+    <!-- header -->
+    <div class="header">
+        <div class="nav">
+            <div class="nav__rightside">
+                <div><h2 class="nav__logo">Medical Core</h2></div>
+                <div class="nav__items">
+                    <a class="nav__item">home</a>
+                    <a class="nav__item" alt="">product</a>
+                    <a class="nav__item" alt="">pricing</a>
+                    <a class="nav__item" alt="">contact</a>
+                </div>
+            </div>
+            <div class="button">
+                <a class="button__linkgreen">login</a>
+                <button class="button__btn--primary">Join us <i class="bi bi-arrow-right-short"></i></button>
+            </div>
+        </div>
+
+        <div class="hero">
+            <p class="hero__title">welcome</p>
+            <h1 class="hero__introduction">A great place to receive care</h1>
+            <p class="hero__subintroduction">Get your best looking fine now</p>
+        <div class="button" >
+        <button class="button__btn--primary">Join us</button>
+        <button class=" button__linkgreen button__linkgreen--border">Learn more</button>
+        </div>
+        </div>
+    </div>
+
+    <!-- main -->
+    <main class="main">
+
+    <div class="container">
+        <div >
+            <img class="container__image" src="./img/thumb-concept.png" alt="">
+        </div>
+        <div class=" container__text container__text--big-gap">
+                <div class="container__border"></div>
+                <div  class="container__heading"><h2>Meet our experts</h2></div>
+                <div class="container__paragraph">
+                    <p >Problems trying to resolve the conflict between the two major realms of Classical physics: Newtonian mechanics </p>
+                </div>
+                <a class="button__linkgreen " href="">Learn more<i class="bi bi-chevron-right"></i></a>
+        
+        </div>
+    </div>
+    <div class="team">
+        <div class="container__text">
+            <p class="container__paragraph--color">team</p>
+            <h2 class="container__heading">Our Team</h2>
```

Comment:
These elements are outside of there block. ( container's elements inside team block)